### PR TITLE
Add Card components

### PR DESCRIPTION
**Pull request by AnkurChunekar**

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
suggestion : add subheading for each card, other than this your code is good to merge

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+.card-icons .card-like-icon .icon:hover {
+  color: red;
+}
+
+/* Card Text Content */
+.card .card-body .card-text-content {
+  margin: 1rem 1.5rem;
+}
+
+.card .card-body .card-text-content .card-description {
```
Comment (kart-c) [inline components/card/card.css]: You should avoid nesting too much where possible

**Review by kart-c — APPROVED** [review]
Looks good Ankur, just one suggestion you can reduce nesting to improve readability of code

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+.card .card-body .card-icons .card-left-badge {
+  position: absolute;
+  top: 2rem;
+  left: -0.2rem;
+}
+.card.card-w-badge .card-body .card-icons .card-like-icon {
+  position: absolute;
+  top: 1.2rem;
+  right: 0.5rem;
+  color: gray;
```
Comment (RakshaPawar108) [inline components/card/card.css]: I see you have used variables for colours except for some places like this one. You can try to make variables wherever possible. Just a suggestion. Great work on the cards though!

**Review by RakshaPawar108 — APPROVED** [review]

File: components/card/card.css (hunk 1 above)
Comment (AnkurChunekar) [reply components/card/card.css]: Okay got it i'll make the changes

File: components/card/card.css (hunk 2 above)
Comment (AnkurChunekar) [reply components/card/card.css]: Okay got it i'll make the changes

**Pull request merged**